packages/osd-i18n/README.md

```diff
@@ -196,7 +196,7 @@ are able to use `FormattedMessage` component in order to translate messages.
 `IntlProvider` should wrap react app's root component (inside each react render method).
 
 In order to translate messages we need to use `I18nProvider` component that
-uses I18n engine under the hood:
+uses I18n engine under the hood. This `I18nProvider` component can use either default or customized locale. If you need a self-defined locale, make sure to register it first using `addLocaleData` method in packages/osd-i18n/src/core/locales.js. Below is an example of how to use `I18nProvider` component:
```

Comment:
Excuse my ignorance, but it may be helpful in making these docs more useful 😂 .

My understanding of the above (and in the context of the rest of the readme) is that:

_To take advantage of `IntlProvider`and `FormattedMessage`, it's not sufficient to simply have a localization file; it's also necessary to make sure the locale that your localization file references is registered with this plugin (via `addLocale`)._

What I think could be made a bit clearer is _when_ a user might need to register a new locale. I see we already have roughly 7 locales registered - so I assume it's only necessary when adding a localization file for a new locale. Is that correct? If so, I think that would be super useful to document as an example. This README currently provides a lot of implementation details, but not really concrete examples of this sort of common task.

I guess where I'm struggling a bit is "can use either default or customized locale" - it's not clear when you'd need a custom or self-defined locale, or when a default would suffice.